addons/binding/org.openhab.binding.pixometer/src/main/java/org/openhab/binding/pixometer/internal/config/ImageMeta.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pixometer.internal.config;
+
+import java.util.List;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link ImageMeta} class is the representing java model for the json result for Image Meta Data from the pixometer
+ * api
+ *
+ * @author Jerome Luckenbach - Initial Contribution
+ *
+ */
+public class ImageMeta {
+
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("annotations")
+    @Expose
+    private List<Annotation> annotations = null;
+    @SerializedName("image")
+    @Expose
+    private String image;
+    @SerializedName("image_download")
+    @Expose
+    private String imageDownload;
+    @SerializedName("camera_model")
```

Comment:
If you use `.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)` when constructing GsonBuilder you can ommit these SerializedName annoations. `cameraModel` will then be auto converted to `camera_model` in json.